local-development/docker-compose.yml

```diff
@@ -44,14 +41,17 @@ services:
       - fairspace-keycloak-db-network
     volumes:
       - ./keycloak/setup-realm.sh:/opt/jboss/startup-scripts/setup-realm.sh
-      - ./keycloak/realm-template.json:/tmp/realm-template.json
-      - ./keycloak/keycloak-add-user.json:/opt/jboss/keycloak/standalone/configuration/keycloak-add-user.json
+      # - ./keycloak/realm-template.json:/tmp/realm-template.json
+      - ./keycloak/realm-export.json:/opt/keycloak/data/import/realm-export.json
```

Comment:
so the realm-export.json replaces realm-template.json? 
I see that now we have both, but only one is used